riak_test/tests/riak_cs_debug_test.erl

```diff
@@ -0,0 +1,170 @@
+%% ---------------------------------------------------------------------
+%%
+%% Copyright (c) 2007-2015 Basho Technologies, Inc.  All Rights Reserved.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% ---------------------------------------------------------------------
+
+-module(riak_cs_debug_test).
+
+-compile(export_all).
+-export([confirm/0]).
+-include_lib("eunit/include/eunit.hrl").
+
+-define(assertContainsAll(ExpectedList,ActualList),
+        lists:foreach(
+          fun(X) -> ?assert(lists:member(X, ActualList)) end,
+          ExpectedList)).
+
+-define(assertMatchAny(Pattern, ActualList),
+        ?assert(
+           lists:any(
+             fun(X) ->
+                     case re:run(X, Pattern) of
+                         {match, _} -> true;
+                         nomatch -> false
+                     end
+             end, ActualList))).
+
+-define(assertNotMatchAny(Pattern, ActualList),
+        ?assert(
+           lists:all(
+             fun(X) ->
+                     case re:run(X, Pattern) of
+                         {match, _} -> false;
+                         nomatch -> true
+                     end
+             end, ActualList))).
+
+confirm() ->
+    %% Run riak-cs-debug before cuttlefish generates configs.
+    TarGz1 = exec_cs_debug(),
+    List1 = trim_dir_prefix(list_files(TarGz1)),
+    ?assertContainsAll(minimum_necessary_files(), List1),
+
+    _ = rtcs:setup(1),
+
+    %% Run riak-cs-debug after cuttlefish generates configs.
+    TarGz2 = exec_cs_debug(),
+    List2 = trim_dir_prefix(list_files(TarGz2)),
+    ?assertContainsAll(minimum_necessary_files_after_boot(), List2),
+    ?assertMatchAny("^logs/platform_log_dir/access.log.*", List2),
+    ?assertMatchAny("^config/generated.configs/app.*.config", List2),
+    ?assertMatchAny("^config/generated.configs/vm.*.args", List2),
+    ?assertNotMatchAny("^config/*.pem$", List2),
+    ok = file:rename(TarGz2, TarGz2 ++ ".back"),
```

Comment:
Agreed. I intentionally avoided to use 'rm' command since it sometimes cause a fatal accident such as `rm -rf / var/hogehoeg` :bomb: But it seems I worried too much. Will change to use 'rm' :chicken: